cedar-policy-validator/src/lib.rs

```diff
@@ -511,3 +777,87 @@ mod test {
         );
     }
 }
+
+#[cfg(feature = "level-validate")]
+#[cfg(test)]
+mod levels_validation_tests {
```

Comment:
Needs some more test cases, particularly if-then-else, and records

Reply:
Added in latest revision